packages/expo-image/expo-image.podspec

```diff
@@ -0,0 +1,21 @@
+require 'json'
+
+package = JSON.parse(File.read(File.join(__dir__, 'package.json')))
+
+Pod::Spec.new do |s|
+  s.name           = 'expo-image'
```

Comment:
I decided to go with `expo-image` since that's where I think React Native libraries are leaning towards (I think the most common way of naming was `RNSomethingSth` and now it has been changed to `react-native-something-sth`).

I agree with the concern that we should figure out the final name before publishing the package. 👍 

When it comes to autolinking I know not all projects are using it, but its rising adoption benefits us (renaming the package doesn't break pod installations).

Reply:
Okay sounds good, lets use the longer name 👍